integrations/http/http.go

```
@@ -233,27 +233,27 @@ func toStruct(r *http.Response) (sdktypes.Value, error) {
 		jsonValue, _ = sdkvalues.Wrap(data)
 	}
 
-	if jsonValue == nil {
-		jsonValue = sdktypes.NewNothingValue()
+	if !jsonValue.IsValid() {
+		jsonValue = sdktypes.Nothing
 	}
 
 	return sdktypes.NewStructValue(
 		sdktypes.NewStringValue("http_response"),
 		map[string]sdktypes.Value{
 			"url":         sdktypes.NewStringValue(r.Request.URL.String()),
 			"status_code": sdktypes.NewIntegerValue(int64(r.StatusCode)),
-			"headers": sdktypes.NewDictValue(
-				kittehs.TransformMapToList(r.Header, func(k string, vs []string) *sdktypes.DictValueItem {
-					return &sdktypes.DictValueItem{
+			"headers": kittehs.Must1(sdktypes.NewDictValue(
```

I don't like using `Must` variants in a code that's not in `init` or `var`. Should return an error.

---

i use `Must`s only if there is no way this can fail. I think it's better than just ignoring the error. Does it make sense?